### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by NikHillAgar**

### CODE REVIEW

**Comment by agshubhs** [comment]
@Nikhil569 Following changes are needed.
1. Directory structure is not followed. Refer guidelines for that
2. Put all the code file inside the folder src.
3. insert images inside the readme file.
4. Do not write everything in the capital letter like Name in a team member section.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,53 @@
...
+For controlling the bot MPU along with the Node mcu was placed on a hand glove.  
+
+## ELECTRONICS ASPECTS   
+Electronics design was basically divided into two parts: Sender and reciever end.  
+At sender, which was mounted on gloves, with purpose of observing gesture. We used MPU-9250, gyroscope for having tilt of hand. It was read by microcontroller Node MCU esp 8266. The SDA and SCL pin of MPU was solded to D2 and D1 respectively, while VCC and ground of MPU were also solded to MCU using male female holders, jumper wires were not used to avoid delay in data which resulted in stoppage of data flow.   
+MPU reading was mapped accordingly and sent to receiver MCU using http server  communication in form of . At the reciever end, reciever MCU use to transfer pwm data to motor driver. This motor driver use to control both the motors accordingly. Input pins were connected to D0 and D1 of MCU and direction pins were connected to D2, D3, D4, D5 respectively.   
+Battery was used to power motordriver bacross 12V and ground terminal, and through 5V and ground terminal we powered MCU. Motors were powereed and controlled through motor driver.
+
+## COST STRUCTURE
+For one bot
```
Comment (agshubhs) [inline README.md]: Make a table for cost structure

File: README.md [hunk 2]
```diff
@@ -1,2 +1,53 @@
...
+
+## MECHANICAL ASPECTS   
+Our design doesn't consists of any complex or advanced parts. That's why we were able to make it without using solidworks.  
+Chasis consists a board(19*16cm,such that circuit components can be placed) on which at back,on left and right side two motors were fixed using L-bracket. In front, at the centre a castor wheel was used at the centre for 360 degree movement.
+The Motor Driver was placed at the center for easy connections. Node Mcu was placed on the left and battery was placed on the right for stability.   
+For controlling the bot MPU along with the Node mcu was placed on a hand glove.  
+
+## ELECTRONICS ASPECTS   
+Electronics design was basically divided into two parts: Sender and reciever end.  
+At sender, which was mounted on gloves, with purpose of observing gesture. We used MPU-9250, gyroscope for having tilt of hand. It was read by microcontroller Node MCU esp 8266. The SDA and SCL pin of MPU was solded to D2 and D1 respectively, while VCC and ground of MPU were also solded to MCU using male female holders, jumper wires were not used to avoid delay in data which resulted in stoppage of data flow.
```
Comment (agshubhs) [inline README.md:19]: Write this in numbered list

**Comment by agshubhs** [comment]
@Nikhil569  Following changes are needed:
1. Readme is incomplete.Many sections are missing refer guildelines for that
[MaRS Documentation(1).pdf](https://github.com/marsiitr/Gesture-Controlled-Bot-1.1/files/3026906/MaRS.Documentation.1.pdf)
2. Mechanical Design folder  missing.

**Comment by agshubhs** [comment]
@Nikhil569 
1. Still, some sections are missing in Readme file.
2. All the three images you have uploaded are similar. Upload different image. Upload one image along with team members and one image of both the bots.
3. Write all the headings normally not in capital letters.

**Comment by NikHillAgar** [comment]
> @Nikhil569
> 
> 1. Still, some sections are missing in Readme file.
> 2. All the three images you have uploaded are similar. Upload different image. Upload one image along with team members and one image of both the bots.

Sir we don't have more images other than these

**Comment by agshubhs** [comment]
> > @Nikhil569
> > 
> > 1. Still, some sections are missing in Readme file.
> > 2. All the three images you have uploaded are similar. Upload different image. Upload one image along with team members and one image of both the bots.
> 
> Sir we don't have more images other than these

use Srishti drive link for the images

**Comment by NikHillAgar** [comment]
> > > @Nikhil569
> > > 
> > > 1. Still, some sections are missing in Readme file.
> > > 2. All the three images you have uploaded are similar. Upload different image. Upload one image along with team members and one image of both the bots.
> > 
> > 
> > Sir we don't have more images other than these
> 
> use Srishti drive link for the images

I have checked it. Couldn't find any.

**Pull request merged**